Sources/Basics/TemporaryFile.swift

```diff
@@ -34,7 +36,6 @@ import TSCBasic
 ///           The cleanup block should be called when the temporary directory is no longer needed.
 ///
 /// - Throws: An error when creating directory and rethrows all errors from `body`.
-@available(macOS 12, iOS 15, tvOS 15, watchOS 8, *)
```

Comment:
Replacing this with `swift(>=5.5.2)` allows us to use these functions with older OS versions.

Reply:
Technically speaking, we can't require that due to Swift CI having 13A5201i installed (which is not even the final 5.5.0).

Reply:
With that Xcode version and `@available(macOS 12, iOS 15, tvOS 15, watchOS 8, *)` applied, these functions aren't used on CI either, are they?